tests/unit/lib/SGVector_unittest.cc

```diff
@@ -181,12 +184,9 @@ TEST(SGVectorTest,complex128_tests)
 	for (index_t i=0; i<res.vlen; ++i)
 		EXPECT_EQ(res[i], i);
 
-	a.scale(complex128_t(1.0));
-	for (index_t i=0; i<a.vlen; ++i)
-	{
-		EXPECT_NEAR(a[i].real(), 10.0, 1E-14);
-		EXPECT_NEAR(a[i].imag(), 12.0, 1E-14);
-	}
+	// removed EXPECT_NEAR block for SFVector::scale
+	// but inserted linalg because further tests are made on data after scaling
+	linalg::scale<linalg::Backend::NATIVE>(a, complex128_t(1.0));
```

Comment:
That's why I wrote this comment. I thought that it will be proper for linalg::scale() to be tested separately as long as it's the unit test of SGVector. Now I get it, thanks!